src/esp/sensor/Sensor.h

```diff
@@ -95,6 +97,16 @@ class Sensor : public Magnum::SceneGraph::AbstractFeature3D {
 
   SensorSpec::ptr specification() const { return spec_; }
 
+  //! Returns the size of the framebuffer corresponding to the sensor's
+  //! resolution
+  Magnum::Vector2i framebufferSize() const {
+    // NB: The sensors resolution is in H x W format as that more cleanly
```

Comment:
Typos:
`sensors` -> `sensor's`
`pratice` -> `practice`
`and DL` -> `and DL.`
`expected` -> `expect`